api/app/models/common/areas_of_interest.py

```diff
@@ -62,32 +62,32 @@ def set_version_default(cls, v):
 
 class KeyBiodiversityAreaOfInterest(AreaOfInterest):
     type: Literal["key_biodiversity_area"] = "key_biodiversity_area"
-    id: str = Field(
-        ..., title="Key Biodiversity Area site code", examples=["36", "18", "8111"]
+    ids: List[str] = Field(
+        ..., title="Key Biodiversity Area site codes", examples=[["36"], ["18", "8111"]]
     )
 
 
 class ProtectedAreaOfInterest(AreaOfInterest):
     type: Literal["protected_area"] = "protected_area"
-    id: str = Field(
+    id: List[str] = Field(
         ...,
-        title="WDPA protected area ID",
-        examples=["555625448", "148322", "555737674"],
+        title="WDPA protected area IDs",
+        examples=[["555625448"], ["148322", "555737674"]],
     )
 
 
 class IndigenousAreaOfInterest(AreaOfInterest):
     type: Literal["indigenous_land"] = "indigenous_land"
-    id: str = Field(
+    id: List[str] = Field(
         ...,
         title="Landmark Indigenous lands object ID",
-        examples=["1931", "1918", "43053"],
+        examples=[["1931"], ["1918", "43053"]],
```

Comment:
I'm a little confused here. Looking at the examples, shouldn't the type be List[List[str]] ?